src/encoded/ingestion_listener.py

```diff
@@ -341,7 +341,6 @@ def __init__(self, vapp, _queue_manager=None, _update_status=None):
                             (self.vapp, _queue_manager, _update_status))
         self.queue_manager = IngestionQueueManager(registry) if not _queue_manager else _queue_manager
```

Comment:
I think this situation is more clearly expressed as (untested):
```suggestion
        self.queue_manager = _queue_manager or IngestionQueueManager(registry)
```